src/mf/mf-hid.cpp

```diff
@@ -442,6 +442,7 @@ namespace librealsense
                                                     auto node = cm_node::from_device_path( propertyValue.pwszVal );
                                                     if( node.valid() )
                                                     {
+                                                        //TODO SAMER
                                                         // We take the "unique id" (really, the composite ID used to associate all the devices belonging to
```

Comment:
@maloel should we change this example comment if we are taking of SKU D465? 
(you gave an example about D465 in the lines below)

Reply:
No, you can leave it if it doesn't bother you :)